tests/unit/resizable/core.js

```diff
@@ -244,4 +244,134 @@ QUnit.test( "nested resizable", function( assert ) {
 	outer.remove();
 } );
 
+QUnit.test( "Resizable with scrollbars and box-sizing: border-box", function( assert ) {
```

Comment:
We have 4 tests here that are largely copy-pasted. I'd consider moving the test body to a generic function accepting the `borderBox` & `transforms` boolean parameters and then use them inside to apply the correct logic & messages.

See an example of a similar pattern in the jQuery repo:
https://github.com/jquery/jquery/blob/4b7ecbad24463c875f03ef4c7a7d307a091f93fd/test/unit/ajax.js#L1821-L1871

jQuery here uses an anonymous function inside the iterators to define the test body. I think I'd prefer to extract it into a parametrized function here to avoid the large indentation, but other than that it looks fine.